cmd/gbounty/bootstrap/bootstrap.go

```diff
@@ -42,6 +42,10 @@ const (
 	debugServerShutdownTime = 5 * time.Second
 )
 
+var (
+	PocEnabled = isPocEnabled()
```

Comment:
If we can get rid of this https://github.com/BountySecurity/gbounty/pull/8/files#r1823611114, we can clean this up, and simplify much the code.

I'd love it, cause this is starting to become a bit spaghetti 😅

Reply:
removed it